common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -257,131 +276,167 @@ div.video {
+                padding-left: lh(.5);
+              }
             }
+          }
 
-            &.pause {
-              background-position: 16px -50px;
-            }
+          .video-control:first-child {
+            border-right: 1px solid #666;
           }
 
-          div.vidtime {
-            font-weight: bold;
-            line-height: 46px; //height of play pause buttons
-            -webkit-font-smoothing: antialiased;
-            padding-left: lh(.75);
-            @media (max-width: 1120px) {
-              padding-left: lh(0.5);
-            }
+          @media (max-width: 1120px) {
+            margin-right: lh(.5);
+            font-size: em(14);
           }
         }
       }
 
-      div.secondary-controls {
-        float: right;
-
-        a.speed-button,
-        div.volume > a,
-        a.add-fullscreen,
-        a.quality-control,
-        a.hide-subtitles {
-          // overflow is used to bypass Firefox CSS :focus outline bug
-          // http://johndoesdesign.com/blog/2012/css/firefox-and-its-css-focus-outline-bug/
-          &:focus {
-            @extend %ui-depth5;
-            position: relative;
-            outline: $white dotted thin;
-            outline-offset: -2px;
-            overflow: auto;
-          }
-        }
+      .secondary-controls {
+        display: block;
+        float: left;
 
-        .menu-container {
-          float: left;
+        .speeds {
+          display: block;
           position: relative;
+          float: left;
+          border-right: 1px solid #666;
 
-          &.is-opened {
-            .menu {
-              display: block;
-              opacity: 1;
-              padding: 0;
-              margin: 0;
-              list-style: none;
+          .speed-button {
+            @extend %video-button;
+            @include clearfix();
+            border: none;
+            text-indent: 0;
+
+            @media (max-width: 1120px) {
+              min-width: 0;
+              width: 60px;
+            }
+
+            .icon {
+              float: left;
+              margin-top: 1em;
+              min-width: ($baseline / 2);
+            }
+
+            .label {
+              float: left;
+              padding: 0 lh(.25) 0 lh(.5);
+              font-size: em(14);
+              font-weight: normal;
+              letter-spacing: 1px;
+              text-transform: uppercase;
+              color: #999;
+
+              @media (max-width: 1120px) {
+                display: none;
+              }
+            }
+
+            .value {
+              float: left;
+              margin-bottom: 0;
+              padding: 0 lh(.5) 0 0;
+              font-weight: bold;
+              color: #fff;
+
+              @media (max-width: 1120px) {
+                padding: 0 lh(.5) 0 lh(.5);
+              }
             }
           }
 
-          .menu {
+          .video-speeds {
             @include transition(none);
-            @extend %ui-depth1;
```

Comment:
Don't think I was the one who removed it, but all but a few properties here were removed. Looks like it was cleaned up? Z-indexing doesn't appear to be an issue so I'll leave it off.